Docker.DotNet/Properties/AssemblyInfo.cs

```diff
@@ -29,5 +29,5 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.16.0.0")]
+[assembly: AssemblyFileVersion("1.16.0.0")]
```

Comment:
@jgarverick hey, I'm trying to understand why this change is needed. :smiley:  finally had time to come back to this repo and release a new version.